config.json

```diff
@@ -184,7 +184,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "features-oop",
+      "title": "Modern",
+      "content": "VB.NET is a modern, fast-evolving language."
```

Comment:
Is the modern and fast-evolving part true of vb.net? I got the feeling that vb.net was progressing relatively slowly, but I might be wrong.

Reply:
Welllll ... I'm sort of assuming that it's fast evolving as it has to keep in step more or less with C#, at least at an "under the hood" level

Reply:
https://docs.microsoft.com/en-us/dotnet/visual-basic/whats-new/#visual-basic-169